components/opponent/wikis/trackmania/opponent_display_custom.lua

```diff
@@ -0,0 +1,223 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:OpponentDisplay/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local DisplayUtil = require('Module:DisplayUtil')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+local PlayerDisplay = Lua.import('Module:Player/Display', {requireDevIfEnabled = true})
+
+local OpponentDisplayCustom = Table.deepCopy(OpponentDisplay)
+
+local html = mw.html
```

Comment:
**imo** kick it
no gain in having this when it is only used 4 times
just write mw.html in the 4 lines where this was used

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/2262/commits/4364489264f630a4bf8c735a01d8eebc8fa390c8